src/tinkerio.py

```diff
@@ -996,6 +1019,10 @@ def molecular_dynamics(self, nsteps, timestep, temperature=None, pressure=None,
         havekeys = set()
         first_shot = True
         for ln, line in enumerate(oanl):
+            if 'Polarization' in line:
```

Comment:
Can you please comment on why you want to skip parsing these lines out of analyze?